### PR TITLE
FOCUS #604: Issue templates review

### PR DESCRIPTION
**Pull request by ijurica**
Upon reviewing the existing five issue templates, I propose keeping the following three:

- **Work Item template:**  
  - Primarily for FOCUS Project members.  
  - If we reach consensus on removing the Discussion template, any necessary changes to this template can be addressed in this PR.

- **Feedback template:**  
  - Primarily for FOCUS User Group members. 
  - To be used for providing feedback for the following areas:
    - Unsupported/missing FinOps features, use cases, or scenarios
    - Existing FOCUS specification content
    - Other
  - See the changed files for specific updates suggested in this PR.

- **Maintenance template:**  
  - Out of scope for this PR.
  - Primarily deals with GitHub-related tasks, the mechanics of generating the spec (PDF and HTML), etc. 

As for the **Spec Change** and **Discussion** templates, I propose we discard both:

- **Spec Change template:**  
  - The group has already discussed this and largely agreed to remove it. Once formal consensus is reached, this PR can include its removal.

- **Discussion template:**  
  - I recommend we discard this template as well. 
  - FOCUS Project members can use the **Work Item** template to introduce discussion topics as potential Work Item candidates or placeholders. These placeholders can be labeled appropriately until fully completed and ready for review and triage by the group. 
  - When and if consensus is reached on this approach, this PR can handle the removal.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feedback.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+## 5. **Criticality Scale**
+Indicate how critical this issue or feature request is for your organization using the scale provided below.
+
+**On a scale of 1 - 4, how critical is this for your organization?**  
+_Select one of the provided options:_  
+- 1: Blocks my organization from adopting FOCUS
+- 2: Important for adoption in the next 3-6 months
+- 3: Important for adoption in the next 6-12 months
+- 4: Suggestion, not planning to use FOCUS soon
```
Comment (ijurica) [inline .github/ISSUE_TEMPLATE/feedback.md:59]: I'm not sure if the Criticality is meant to indicate 
- the urgency of resolving the issue 
- or the organization’s own FOCUS adoption plans. 

Assuming we're focusing on the urgency of resolution, I propose the following suggestion:

```suggestion
- 1: Critical - Blocks my organization from adopting FOCUS
- 2: Important - Ideally resolved within the next 3-6 months
- 3: Important - Ideally resolved within the next 6-12 months
- 4: Suggestion - Resolution not urgent
```

Comment (ahullah) [reply .github/ISSUE_TEMPLATE/feedback.md:59]: Agree with suggested change

File: .github/ISSUE_TEMPLATE/feedback.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+# Feature Request and Use Case Feedback
```
Comment (ijurica) [inline .github/ISSUE_TEMPLATE/feedback.md:1]:
```suggestion
# Feedback on Unsupported Features, Use Cases, and Existing FOCUS Spec.
```

**Comment by jpradocueva** [comment]
### Action Items from TF-1 call on Oct 22:
- [ ] [[#612](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/612)] Irena @ijurica : Commit display adjustments to the Markdown version for clarity and ensure final formatting changes.
- [ ] [[#612](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/612)] All members: Review the changes in the feedback template and provide feedback on the criticality section suggestions.

**Comment by jpradocueva** [comment]
### Action Items Members' call on Oct 31:
- [ ] [[#612](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/612)] Irena @ijurica : Collect and implement feedback on the current markdown version for further testing.
- [ ] [[#612](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/612)] All members: Review and provide feedback on the current Markdown issue templates before the next meeting to finalize improvements.

File: .github/ISSUE_TEMPLATE/feedback.yml [hunk 3]
```diff
@@ -1,48 +1,114 @@
...
   - type: markdown
     attributes:
-      value: "FOCUS working group seeks gaps in the current specification for FinOps and beyond. Share unsupported use cases or features to prioritize updates. Avoid sharing proprietary information."
-  
+      value: |
+        ---
+        ### **Template Usage Notes**:
+        1. All fields marked as **mandatory** [*] must be filled before submission.
+        2. For **Supporting Data/Documentation**, ensure that linked files are accessible to relevant stakeholders.
+        3. If you wish to file a bug report using the markdown version of this issue template, please click [here](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/issues/new?template=feedback.md)
```
Comment (jpradocueva) [inline .github/ISSUE_TEMPLATE/feedback.yml:17]:
```suggestion
```

**Comment by jpradocueva** [comment]
Approved during the Members' call on Nov 7.

**Comment by jpradocueva** [comment]
### Action Items from Members' meeting on November 7:
- [ ] [[#612](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/612)] All members: Review the new templates in the test environment to ensure all required sections and workflows function as expected.

**Review by shawnalpay — APPROVED** [review]

**Pull request merged**